core/libqutim/chatsession.cpp

```diff
@@ -211,11 +231,18 @@ ChatLayer::ChatLayer() : d_ptr(new ChatLayerPrivate)
 	                                QLatin1String("SenderHook"),
 	                                MessageHandler::NormalPriortity,
 	                                MessageHandler::SenderPriority);
+
+	MessageHandler::registerHandler(p()->historyHook.data(),
+									QLatin1String("HistoryHook"),
+									MessageHandler::HighPriority + 100500 - 1,
+									MessageHandler::HighPriority + 100500 - 1);
```

Comment:
May be you may introduce MessageHandler::HistoryPriority enum value? I believe it may be useful for information for plugins